samples/js/picasa/find-photos/src/main.js

```diff
@@ -172,10 +174,19 @@ function getGeo(input): ?Struct {
   });
 }
 
-async function getSizes(input: Object, origWidth: number, origHeight: number)
+async function getSizes(resources: Map<Struct, Struct>)
     : Promise<Map<Struct, string>> {
+  const tuples = [];
+  await resources.forEach((v: any, k: Struct) => {
+    tuples.push([k, v.url]);
+  });
+  return Promise.resolve(new Map(tuples));
```

Comment:
Same thing as above - when the function is `async`, the `Promise` wrapper is done automatically.